app/atomic_api.py

```diff
@@ -0,0 +1,28 @@
+import json
+import logging
+from aiohttp import web
+from aiohttp_jinja2 import template
+
+from app.utility.base_service import BaseService
+from app.service.auth_svc import for_all_public_methods, check_authorization
+from plugins.atomic.app.atomic_svc import AtomicService
+
+
+@for_all_public_methods(check_authorization)
+class AtomicAPI(BaseService):
+
+    def __init__(self, services, name, description):
+        self.services = services
```

Comment:
Is `self.services` used beyond setting `auth_svc` and `data_svc`? That is, does it need to be an instance variable? I know that `self.auth_svc` is required for the `check_authorization` function to work, but I don't know if anything requires that.